templates/single.twig

```diff
@@ -36,29 +36,11 @@
 		<!-- Post Block Start -->
 		<div class="container">
 			<div class="post-content">
+				{% if ( post.take_action_boxout ) %}
+					{{ post.take_action_boxout|shortcodes|raw }}
```

Comment:
Do we need escaping here together with the |raw filter? Or is it already escaped?

Reply:
No its is not escaped. I think we will need escaping here. 
Good catch, I will update the code.

Reply:
Actually escaping here will remove style attribute from top level element in the take action action boxout block (no background image).
I will add escaping in the take action boxout block itself, so I think we are good to go with this.

Reply:
ok